app/Models/User.php

```diff
@@ -26,4 +26,9 @@ class User extends Authenticatable
     protected $hidden = [
         'password', 'remember_token',
     ];
+
+    public function match() {
+
+        return $this->hasMany('App\Models\Match');
```

Comment:
this is a many relationship so the name has to be plural (matches) also this needs to be a belong to many - https://laravel.com/docs/5.4/eloquent-relationships#many-to-many - and you need a pivot table (read the article and we can discuss)